src/core_modules/capture-core/components/Pages/common/TEIRelationshipsWidget/TeiSearch/TeiSearchForm/TeiSearchForm.component.js

```diff
@@ -130,7 +130,7 @@ class SearchFormPlain extends React.Component<Props, State> {
 
     getUniqueSearchButtonText = (searchForm) => {
         const attributeName = searchForm.getElements()[0].formName;
-        return `Search ${attributeName}`;
+        return `${i18n.t('Search')} ${attributeName}`;
```

Comment:
To fix the regular Search page too, could we apply the same fix to `src/core_modules/capture-core/components/TeiSearch/TeiSearchForm/TeiSearchForm.component.js#L134`? Thanks!